gen/__init__.py

```diff
@@ -306,20 +305,82 @@ def do_gen_package(config, package_filename):
     log.info("Package filename: %s", package_filename)
 
 
-def extract_files_with_path(start_files, paths):
+def render_late_content(content, late_values):
+    # TODO(branden): Unit tests
```

Comment:
It's removed when unit tests are added at https://github.com/dcos/dcos/pull/1163.